docs/MPQEditor.md

```diff
@@ -0,0 +1,63 @@
+# MixedPrecisionQantizationEditor (MPQEditor for short)
+
+## About
+
+MPQEditor povides easy way of creation/editing MPQ json files, suitable for Quantization step in CfgEditor. Also it gives the ability to select the most sensitive to quantization layers using `visq` data. 
+
+## How to use
+
+Currently MPQEditor is integrated into EXPLORER view by context menu item "Create MPQ json".
+
+In order to use it:
+1) call context menu for a `.circle model` in EXPLORER view
+2) call "Create MPQ json"
+3) give a name to MPQ json
+4) edit MPQ json using MPQEditor
+5) all layers from 'layers' section in json are shown in grid-table, so that their options for quantization can be edited using their respective dropdowns
+6) to make 'model-graph-view' more informative the user may use 'visq' information, it shows which layers are more sensitive to quantization than others
+
+<img src="./images/MPQ_UI.png"/>
+
+## Implementation notes 
+By default `model-graph-view` is shown in `selector` mode to provide the way to select multiple nodes.
+When the user loads `visq` file, we need a way for fast resetting `__viewMode` of `index.html`, that is why new message `changemode` was introduced.
+Currently `viewMode.visq` can not be used, because it supports only single selection and for selected item `properties` panel is called.
+We need such a mode, that has the ability of showing `visq` colormaps along with the ability to select multiple nodes.
+That is why new `viewMode.visqselector` mode was introduced.
+
+## FAQ
+
+MPQEditor provides a convenient way to edit configuration file suitable for mixed precision quantization.
+
+> what is the purpose of this?
+
+
+The purpose of MPQEditor is in providing convenient way of editing json file, which can be used later in quantization step for
+mixed precision quantization.
+
+> what does the user get from this?
+
+Instead of text-mode editing 'layers' section in MPQ json file the user gets the tool to edit 'layers' section using 'graph-model-view' and to select the most sensitive layers depending on their sensitivity.
+
+> what does the user have to do with the result?
+
+
+The user may use the resulting json for Quantization step in CfgEditor.
+
+> when does the mode change to visqselector?
+
+
+When the user loads `visq` file with errors of quantization
+
+> what do this program expect from visqselector?
+
+
+The program expects the ability to select multiple layers along with these `visq` colormaps to give the user a hint which layers are more sensitive to quantization than others.
+
+> how does the user get back to this?
+
+
+The user can empty `visq` input.
```

Comment:
this UI is implicit. it would be better to provide explicit way of turning on and off visq.json file.

Reply:
ok.